app/pages/nonsipp/memberpensionpayments/MemberPensionPaymentsListPage.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.nonsipp.memberpensionpayments
+
+import models.SchemeId.Srn
+import pages.QuestionPage
+import play.api.libs.json.JsPath
+
+case class MemberPensionPaymentsListPage(srn: Srn) extends QuestionPage[Boolean] {
```

Comment:
Just as a matter of interest.... if this is not saved, would this still work without a path and toString, extending Page? As for example: `WhatYouWillNeedLoansPage`?